crates/cli/src/main.rs

```diff
@@ -504,6 +521,9 @@ mod windows {
         fn run_foreground(&self, _ipc_url: String) -> io::Result<ExitStatus> {
             unimplemented!()
         }
+        async fn get_env_vars(&self) -> Option<HashMap<String, String>> {
+            unimplemented!()
```

Comment:
* Why do we need a whole new trait method just for the `cli`? (breaking Windows even more, along the way).
Cannot CLI call whatever `util` methods it needs on the spot, without all this?

* What does `None` even mean in this context and why none of the impls return that?

Reply:
- We can very well do that, I just preferred not to handle `cfg` checks manually.
- My bad, it can't be `None`.

Reply:
Let's do that then, adding some odd public API method is much worse any manual config checks IMO.